scripts/workers/code_upload_submission_worker.py

```diff
@@ -368,7 +557,15 @@ def main():
     while True:
         message = evalai.get_message_from_sqs_queue()
         message_body = message.get("body")
+        message_body["submission_time_limit"] = challenge.submission_time_limit
```

Comment:
Why are we passing `submission_time_limit` in sqs message? This will be a fixed value for each challenge. Let's pass this as task_definition param to the worker. Let's not add too many details in sqs message

Reply:
@Ram81, Since we are getting challenge object already in code upload worker, why is there a need to pass it from task_defn_worker? 
We could just set an env variable one time at the starting of the code upload worker when we are fetching the challenge details.
OR 
We could have a variable `submission_time_limit` passed explicitly to the function.

The submission_time_limit is not in the actual sqs message though it is in our local message_body variable but yeah we have that separate for good abstraction. The disadvantage is when we have some more info to pass we eventually have to pass it in some dict.
What do you suggest how we proceed?

Reply:
I see, let's create a new dict `submission_meta` and populate that in the same message body object for now. Adding new param to existing functions will need more changes.

Reply:
Updated these changes.